### ISSUE TITLE
VB misidentifying invocation as unused property access

### ISSUE THREAD

**Issue by JoeRobich** [dotnet/roslyn#29105]
**Version Used**: 
Visual Studio 15.8 Preview

**Steps to Reproduce**:
Place the following in a VB editor.
```vb
Class C
    Public Property A As Action
End Class

Module Program
    Sub Main()
        Dim c As C = New C()
        c.A()
    End Sub
End Module 
```

**Expected Behavior**:
No Warnings

**Actual Behavior**:
The compiler is flagging `c.A()` with "Property access must assign to the property or use its value"

**Comment by JoeRobich** [comment]
The invocation was on the property getter itself not an invocation of the Action.
